notify_python35.cpp

```diff
@@ -219,6 +221,21 @@ bool NotifyPython35::reconfigure(const std::string& newConfig)
 	// Configuration change is protected by a lock
 	lock_guard<mutex> guard(m_configMutex);
 
+	if (Py_IsInitialized)
```

Comment:
This should be a function call, I think just checking "Py_IsInitialized" would just check the address of this function and that would be non-zero always in this code.